lib/pagseguro/refund.rb

```diff
@@ -12,8 +12,8 @@ class Refund
     attr_accessor :value
 
     # Calls the PagSeguro webservice and register the refund.
+    # Return boolean.
     def register
-      params = Serializer.new(self).to_params
       Response.new(Request.post("transactions/refunds", api_version, params))
```

Comment:
@dmitryrck it is expected to return an instance of refund here with result (ok) or errors.